.github/workflows/ci.yaml

```
@@ -0,0 +1,95 @@
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    if: ${{ !github.event.pull_request.draft }} # only run on non-draft PRs
+
+    steps:
+      - name: Checkout sdk-py
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)